src/wazuh_modules/wm_syscollector.c

```diff
@@ -99,6 +97,8 @@ void* wm_sys_main(wm_sys_t *sys)
     }
     if (syscollector_start_ptr) {
         mtinfo(WM_SYS_LOGTAG, "Starting Syscollector.");
+        syscollector_sync_max_eps = sys->sync.sync_max_eps;
+        mtinfo(WM_SYS_LOGTAG, "MNB: sync_max_eps vale: %ld", syscollector_sync_max_eps);
```

Comment:
remember delete this

Reply:
Will do as soon as I validate the complete e2e behavior. Thanks for remind me :)